### PR TITLE
Add first two sections: header and headline

### PR DESCRIPTION
**Pull request by gfrancoarq**
### I've added the header and headline section. It includes content, assets, and styling.
* In addition to the HTML and CSS file, I've included the README file and MIT license.
* These changes complete the "Portfolio: setup and mobile first" activity

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @gfrancoarq, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a project that has failing linters, Kindly make sure your linters are all passing before submitting the PR.

## Stlylint AutoFix
- [ ] if you have a lot of CSS line errors, I kindly suggest you fix them with `npx stylelint --fix "**/*.{css,scss}"`, but please get used to fixing the errors line by line also because it'll help you learn best practices.

## ESlint AutoFix
- [ ] if you have a lot of ESlint line errors, I kindly suggest you fix them with `npx eslint . --fix`, but please get used to fixing the errors line by line also because it'll help you learn best practices.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

**Comment by gfrancoarq** [comment]
Thank you! I made the corrections according to the observations of linters

File: index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          </div>
+          <div class="scroller-icon">
+            <img
+              src="images/btn-down.svg"
+              alt="down button"
+              class="down-button"
+            />
+          </div>
+        </div>
+      </div>
```
Comment (mahbubul14) [inline index.html:56]: - [ ] You should wrap this section with a `section` tag here. :+1:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Franco's Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <!-- Toolbar -->
+      <nav>
+        <div>
+          <h1>Welcome</h1>
```
Comment (mahbubul14) [inline index.html:15]: - [ ] Kindly use the other` hx` tag instead of `h1` here. :+1:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            />
+          </div>
+          <div class="scroller-icon">
+            <img
+              src="images/btn-down.svg"
+              alt="down button"
+              class="down-button"
+            />
+          </div>
+        </div>
```
Comment (mahbubul14) [inline index.html:55]: - [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (a tag). :+1:

File: README.md [hunk 4]
```diff
@@ -1,56 +1,20 @@
...
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+- HTML and CSS
```
Comment (mahbubul14) [inline README.md:10]: - [ ] In your `README` file, there is no "Getting Started" section. Kindly add the getting started section. This should be an example of how you may give instructions on setting up your project locally. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @gfrancoarq ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

- [ ] The commit message should be written in the imperative tense. `Ex: "Fix bug" instead of "bug fixing".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by zhadier — APPROVED** [review]
## Hi @gfrancoarq  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



# Congratulations! 🎉 

✔️ No linter errors
✔️ Professional pull request title and description

### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**